symfony-app.docker-hybrid/docker-compose.yaml.tmpl

```diff
@@ -84,3 +88,15 @@ services:
       TZ: {{ .Vars.system.timezone }}
 
 {{- end }}
+
+
+{{- if $has_rabbitmq }}
+
+  rabbitmq:
+    image: 'rabbitmq:alpine'
```

Comment:
Utiliser l'image avec le managment intégré (`management-alpine`), sinon pas de web UI

https://stackoverflow.com/questions/47290108/how-to-open-rabbitmq-in-browser-using-docker-container